app/src/main/java/org/mozilla/focus/search/CustomSearchEngineStore.kt

```diff
@@ -34,7 +34,7 @@ object CustomSearchEngineStore {
 
     fun addSearchEngine(context: Context, engineName: String, searchQuery: String): Boolean {
         // This is not for a homescreen shortcut so we don't want an adaptive launcher icon.
-        val iconBitmap = IconGenerator.generateLauncherIconPreOreo(context, engineName[0])
+        val iconBitmap = IconGenerator.generateSearchEngineIcon(context)
```

Comment:
I think we can remove the comment above now since we're not using the launcher icon method?